explore/marginal-gain/1.Marginal-Gain-Multiple-Mutations.ipynb

```diff
@@ -0,0 +1,1299 @@
+       "2     4221   MEN1       0.801961        0.827485       0.775454            11   \n",
+       "4     5979    RET       0.832704        0.790473       0.890186             4   \n",
+       "7     7157   TP53       0.937524        0.946046       0.936537           259   \n",
+       "0     7428    VHL       0.980185        0.995218       0.994318           168   \n",
+       "\n",
+       "   n_neg_coeffs  cum_rank_cov_feat  median_rank_cov_feat  mean_rank_cov_feat  \\\n",
+       "5             0               2415                  34.5           34.500000   \n",
+       "3             0               2415                  34.5           34.500000   \n",
+       "6             0               2415                  34.5           34.500000   \n",
+       "1             0               2415                  34.5           34.500000   \n",
+       "2            28               2415                  34.5           34.500000   \n",
+       "4            18               2415                  34.5           34.500000   \n",
+       "7            28               2415                  34.5           34.500000   \n",
+       "0             5               2415                  34.5           34.500000   \n",
+       "5            84              87223                1409.5         1246.042857   \n",
+       "3             0              93972                1387.5         1342.457143   \n",
+       "6             1              92625                1389.5         1323.214286   \n",
+       "1             1              92143                1389.5         1316.328571   \n",
+       "2             7              92312                1391.5         1318.742857   \n",
+       "4             0              94141                1388.5         1344.871429   \n",
+       "7           179              67199                1456.5          959.985714   \n",
+       "0           109              89046                1433.5         1272.085714   \n",
+       "\n",
+       "             model  \n",
+       "5  covariates_only  \n",
+       "3  covariates_only  \n",
+       "6  covariates_only  \n",
+       "1  covariates_only  \n",
+       "2  covariates_only  \n",
+       "4  covariates_only  \n",
+       "7  covariates_only  \n",
+       "0  covariates_only  \n",
+       "5         combined  \n",
+       "3         combined  \n",
+       "6         combined  \n",
+       "1         combined  \n",
+       "2         combined  \n",
+       "4         combined  \n",
+       "7         combined  \n",
+       "0         combined  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "auroc_dfs['model a']['model'] = 'covariates_only'\n",
+    "auroc_dfs['model b']['model'] = 'combined'\n",
+    "auroc_df = pd.concat([auroc_dfs['model a'],auroc_dfs['model b']])\n",
+    "display(auroc_df)\n",
+    "auroc_df.to_csv(\"./auroc_df.tsv\", sep=\"\\t\")"
```

Comment:
Done. I uploaded the updated notebook and script. Ignore the output (have not rerun) but let me know if there is anything else you'd like me to change before I run tonight.